validator/specs/lib/validateTilesetSpec.js

```diff
@@ -131,6 +164,14 @@ describe('validateTileset', function() {
             }), done).toResolve();
     });
 
+    it('returns error message when a content\'s box type boundingVolume is not within it\'s tile\'s box type boundingVolume', function(done) {
+        var tileset = clone(sampleTileset2, true);
+        expect(validateTileset(tileset)
+            .then(function(message) {
+                expect(message).toBe('content box is not within tile box');
+            }), done).toResolve();
+    });
+
```

Comment:
Can you also add a second test with two OBBs that are not aligned? Also add a test that checks a valid tileset.

It may be worth creating a helper function to generate the tileset based on a provided `boundingVolume` and `contentBoundingVolume` that all these tests can use (and https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/109/files too).